backend/remote/backend.go

```diff
@@ -591,7 +591,7 @@ func (b *Remote) DeleteWorkspace(name string) error {
 }
 
 // StateMgr implements backend.Enhanced.
-func (b *Remote) StateMgr(name string) (state.State, error) {
+func (b *Remote) StateMgr(name string) (statemgr.Full, error) {
```

Comment:
Sneaking in an unrelated change to start removing references to the deprecated `state` package:

https://github.com/hashicorp/terraform/blob/682440766fcc4382a5ed39aa2ea36583a5548419/state/state.go#L16